SConstruct

```diff
@@ -523,7 +523,7 @@ if env["PLATFORM"] != "win32" :
 	env.Append( CXXFLAGS = [ "-std=$CXXSTD", "-fvisibility=hidden" ] )
 
 	if env["BUILD_TYPE"] == "DEBUG" :
-		env.Append( CXXFLAGS = ["-g", "-O0", "-DTBB_USE_DEBUG=1"] )
+		env.Append( CXXFLAGS = ["-g", "-O0", "-DTBB_USE_DEBUG=1"] + ( ["--compress-debug-sections=zlib"] if "clang++" in os.path.basename( env["CXX"] ) else ["-gz=zlib"] ) )
```

Comment:
My version of Clang (Apple 13.1.6) doesn't have the `--compress-debug-sections` flag, but does have the `-gz` one. Can we use `-gz` for both GCC and Clang, or is it dropped from later Clang versions?